pkg/build/graph/analysis/bc.go

```diff
@@ -238,6 +238,19 @@ func FindPendingTags(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 	return markers
 }
 
+// getImageStreamTagSuggestion will return the appropriate marker Suggestion for when a BuildConfig is missing its input ImageStreamTag;  in particular,
+// it will determine whether or not another BuildConfig can produce the aforementioned ImageStreamTag
+func getImageStreamTagSuggestion(g osgraph.Graph, f osgraph.Namer, tagNode *imagegraph.ImageStreamTagNode, bcNode *buildgraph.BuildConfigNode) osgraph.Suggestion {
+	bcs := []string{}
+	if inputBCNode, points := buildPointsToTag(g, tagNode); points {
+		bcs = append(bcs, f.ResourceName(inputBCNode))
```

Comment:
duh ... yep, my thought process only got to the first level ... I'll rework the attempt to get all the bc's

on the name, since `git blame` says you originally introduced the `buildPointsToTag` method @kargakis , that certainly simplifies the debate on what to name it to ;-) ... i'll switch to getBuildsForTag and change both the original call to it in `FindPendingTags` and the one I added in `getImageStreamTagSuggestion` for `FindMissingInputStreams`